index.css

```diff
@@ -87,3 +87,12 @@ button {
   box-shadow: var(--box-shadow-lg);
   transition: all 0.3s ease;
 }
+
+@media screen and (max-width: 640px) {
+  .mobile-hidden {
+    display: none;
+  }
+  .mobile-hidden {
```

Comment:
Look back over the solution on Code Monster for this styling challenge to ensure that you have captured all of the code changes needed to make the page responsive.